docs/cheaha/open_ondemand/ood_jupyterlab.md

```diff
@@ -0,0 +1,65 @@
+# JupyterLab
+
+As described on JupyterLab's [documentation page](https://jupyterlab.readthedocs.io/en/latest/) "JupyterLab is a highly extensible, feature-rich notebook authoring application and editing environment, and is a part of [Project Jupyter](https://docs.jupyter.org/en/latest/), a large umbrella project centered around the goal of providing tools (and [standards](https://docs.jupyter.org/en/latest/#sub-project-documentation)) for interactive computing with computational notebooks. A [computational notebook](https://docs.jupyter.org/en/latest/#what-is-a-notebook) is a shareable document that combines computer code, plain language descriptions, data, rich visualizations like 3D models, charts, graphs and figures, and interactive controls. A notebook, along with an editor like JupyterLab, provides a fast interactive environment for prototyping and explaining code, exploring and visualizing data, and sharing ideas with others."
+
+To launch a JupyterLab session, visit [Cheaha's Open On Demand home page](https://rc.uab.edu/), and select the menu items 'Interactive Apps -> JupyterLab'. The job creation and submission form appears:
```

Comment:
Please use double quotes instead of single quotes to be consistent with other menu item documentation.